parking_permits/models/order.py

```diff
@@ -59,6 +66,61 @@ def validate_order(order_id, user_id):
             raise OrderValidationError("Talpa order is not valid")
 
 
+class SubscriptionValidator:
+    @staticmethod
+    def validate_subscription(user_id, subscription_id, order_id, order_item_id):
+        headers = {
+            "api-key": settings.TALPA_API_KEY,
+            "namespace": settings.NAMESPACE,
+            "user": user_id,
+            "Content-Type": "application/json",
+        }
+        response = requests.get(
+            urljoin(
+                settings.TALPA_ORDER_EXPERIENCE_API,
+                f"subscriptions/get-by-order-id/{order_id}",
+            ),
+            headers=headers,
+        )
+        if response.status_code == status.HTTP_200_OK:
```

Comment:
Does this have to always be 200? i.e. check `response.ok` or call/catch `response.raise_for_status()`.

Reply:
Will change this to `response.ok`